students/slupek_bartosz/lesson_03_functions/the_length_of_the_segment.py

```diff
@@ -0,0 +1,12 @@
+from math import sqrt
+
+
+def length_of_segment(x1, y1, x2, y2):
+    try:
+        length = sqrt((x2 - x1) ** 2 + (y2 - y1) ** 2)
+    except ValueError:
```

Comment:
jeżeli uzytkownik poda string to porgram sie wywali zanim wywoła tą funkcję, ponieważ nie dojdzie do konwersji `float(input)` w linijce 11, wiec lepiej w tamtym miejscu było dodac łapanie wyjątku.